e2e/tests/functional/plugins/timer/timer.e2e.spec.js

```diff
@@ -144,7 +141,7 @@ test.describe('Timer with target date @clock', () => {
 async function triggerTimerContextMenuAction(page, timerUrl, action) {
   const menuAction = `.c-menu ul li >> text="${action}"`;
   await openObjectTreeContextMenu(page, timerUrl);
-  await page.locator(menuAction).click();
+  await await page.locator(menuAction).click();
```

Comment:
i think one of the autofix rules is broken... need to open an issue for this. it did double awaits for me too